provider_base/services/couchdb.json

```diff
@@ -8,8 +8,8 @@
         }
     },
     "couch": {
-        "master": false,
         "port": 5984,
+        "mode": "multimaster",
```

Comment:
As couch 2.0 is far away, and we don't support bigcouch anymore, i would prefer to have 'plain' as default.